src/namespace.js

```diff
@@ -1,6 +1,8 @@
 import $ from 'jquery'
 import { version } from '../package.json'
 
+import { imageLoader, videoLoader } from './utils/image-loader'
```

Comment:
'./utils/image-loader' – maybe better 'media-loader'? because image and video and it would be more common

Reply:
I think it's better now to save the original structure and naming. And make refactoring in the further.